staging/src/k8s.io/apimachinery/pkg/runtime/serializer/protobuf_extension.go

```diff
@@ -21,20 +21,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer/protobuf"
 )
 
-const (
-	// contentTypeProtobuf is the protobuf type exposed for Kubernetes. It is private to prevent others from
-	// depending on it unintentionally.
-	// TODO: potentially move to pkg/api (since it's part of the Kube public API) and pass it in to the
-	//   CodecFactory on initialization.
-	contentTypeProtobuf = "application/vnd.kubernetes.protobuf"
-)
-
 func protobufSerializer(scheme *runtime.Scheme) (serializerType, bool) {
```

Comment:
Wait, I removed this file.  Oh wait, it hasn't merged yet.  Please cherry-pick my change in #76054 that removes this file.

Reply:
Done